addon/components/universal-selection/option/component.js

```diff
@@ -0,0 +1,22 @@
+import Component from '@ember/component';
+import layout from './template';
+import { computed } from '@ember/object';
+
+export default Component.extend({
+  tagName: 'li',
+  classNames: ['ember-power-select-option', 'power-select-option'],
```

Comment:
it's not safe naming your classes like this because the risk of name clash with the `power-select` package is too high. You are not extending a power-select feature here, so universal-selection option class should just be something like `universal-selection-option`.